packages/smart-wallet/package.json

```
@@ -26,6 +26,7 @@
   "dependencies": {
     "@agoric/assert": "^0.6.1-u11wf.0",
     "@agoric/casting": "^0.4.3-u13.0",
+    "@agoric/deploy-script-support": "^0.10.3",
```

On the release branch, this is the version of `deploy-script-support`.

```suggestion
    "@agoric/deploy-script-support": "^0.10.4-u13.0",
```

---

done